.github/workflows/acceptance.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         env:
           GITHUB_PR_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
           GITHUB_PR_REF: ${{ github.event.pull_request.head.ref || github.ref }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
```

Comment:
Can you see if this works without this line? This should already be an environment variable: https://docs.github.com/en/actions/learn-github-actions/environment-variables